boards/px4/fmu-v6c/src/i2c.cpp

```diff
@@ -36,5 +36,20 @@
 constexpr px4_i2c_bus_t px4_i2c_buses[I2C_BUS_MAX_BUS_ITEMS] = {
 	initI2CBusExternal(1),
 	initI2CBusExternal(2),
-	initI2CBusInternal(4),
+	initI2CBusExternal(4),
 };
+
+#ifdef BOARD_OVERRIDE_I2C_TREAT_SENSOR_AS_INTERNAL
+bool i2c_treat_sensor_as_internal(uint32_t device_id)
```

Comment:
Keep the px4_i2c_ prefix naming convention?

Reply:
Ok.